packages/cli/src/index.ts

```diff
@@ -431,40 +430,41 @@ yargs
   )
   .command(
     'build-docs [api]',
-    'build definition into an HTML file',
+    'Produce API documentation as an HTML file',
     (yargs) =>
       yargs
         .positional('api', { type: 'string' })
         .options({
           o: {
-            describe: 'Output file',
+            describe: 'Output destination file.',
             alias: 'output',
             type: 'string',
             default: 'redoc-static.html',
           },
           title: {
-            describe: 'Page Title',
+            describe: 'Page title.',
             type: 'string',
           },
           disableGoogleFont: {
-            describe: 'Disable Google Font',
+            describe: 'Disable Google fonts.',
             type: 'boolean',
             default: false,
           },
           t: {
             alias: 'template',
-            describe: 'Path to handlebars page template, see https://git.io/vh8fP for the example',
+            describe: 'Path to handlebars page template, see https://git.io/vh8fP for the example.',
             type: 'string',
           },
           templateOptions: {
             describe:
-              'Additional options that you want pass to template. Use dot notation, e.g. templateOptions.metaDescription',
+              'Additional options to pass to the template. Use dot notation, e.g. templateOptions.metaDescription',
```

Comment:
```suggestion
              'Additional options to pass to the template. Use dot notation, for example, templateOptions.metaDescription',
```

Reply:
I think we should avoid the expansion here, the content will be displayed on the terminal as help output, so shorter is better, or it causes wrapping output.